### ISSUE TITLE
[API Proposal]: Add ReadOnlySequence<char> overload for WriteRawValue

### ISSUE THREAD

**Issue by Tragetaschen** [dotnet/runtime#102491]
### Background and motivation

In #68223, two overloads for `Utf8JsonWriter` were proposed, one for `ReadOnlySequence<byte>` and one for `ReadOnlySequence<char>`. The latter was rejected due to the necessary surrogate pair state handling while writing the  `ReadOnlySpan<char>` segments.

However, #67337 is currently underway to add the building blocks and state handling for that. Adding the rejected overload now seems like an easy addition.

### API Proposal

```diff
namespace System.Text.Json
{
    public sealed partial class Utf8JsonWriter
    {
+        public void WriteRawValue(ReadOnlySequence<char> json, bool skipInputValidation = false);
    }
}
```

### API Usage

I could avoid building an entire in-memory, contiguous string just to have it written out as JSON. Other proposals like #97570 or #87362 would tie in very nicely as well.

**Comment by Tragetaschen** [comment]
#67377 is implemented. Where can I send the cake for bribery?

**Comment by teo-tsirpanis** [comment]
Duplicate of #68223; this API is available since .NET 8.

**Comment by Tragetaschen** [comment]
This is wrong. The requested overload is the one specifically dropped during review. There is only a `byte` overload, not one for `char`.
https://github.com/dotnet/runtime/issues/68223#issuecomment-1216955662
